website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as a database engine.
+You need to have a running Kubernetes cluster to set up PostgreSQL with DocumentDB extension on Kubernetes.
+
+You can deploy PostgreSQL with DocumentDB extension using any of our provided images.
+Please see the [Docker installation docs](../documentdb/docker.md) to learn more about the available images.
+
+:::tip
+We strongly recommend specifying the full image tag (e.g., `17-0.102.0-ferretdb-2.1.0`)
+to ensure consistency across deployments.
+For more information on the best FerretDB image to use, see the [DocumentDB release notes](https://github.com/FerretDB/documentdb/releases/).
+:::
+
+The following `postgres.yaml` file creates a PostgreSQL instance with the DocumentDB extension:
+
+```yaml
+apiVersion: v1
+kind: Pod
```

Comment:
It's a bit strange that we provide a manifest for a pod instead of StatefulSet.

Reply:
Any reason why @Fashander ?